keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -576,6 +579,71 @@ required elsewhere:
   future allow adding additional data without having to change code
   other than that which uses the new information.
 
+#### Implementation details of using User Namespace to Toggle Ephemeral-Storage-Quotas
+
+* The implementation centers around a feature toggle which 
```

Comment:
Can we describe in more detail why we need to enable user namespaces?

Reply:
Added the description. Please let me know if it sounds good